src/util/linkify.js

```diff
@@ -1,28 +1,34 @@
 
+export const URL_REGEX = /((((ht|f){1}(tps?:[/][/]){1}))[-a-zA-Z0-9@:;%_\+.~#?&/=]+)/gm;
+
+/**
+ * Generates HTML link tag element with target
+ *
+ * @method createLink
+ * @param {Object} options optional modifiers
+ * @param {String} href string of link passed through options
+ * @return {String} The HTML link tag
+ */
+const createLink = options => href => {
+	const target = options.target || '';
+	const targetAttr = `target="${target}"`;
+	const relAttr = target === '_blank' ? ' rel="noopener noreferrer"' : '';
```

Comment:
nitpick, but for consistency, don't put the extra space before `rel` here - put it in between `targetAttr` and `relAttr` in the the string template in the `return` statement. Even if it's an empty string HTML won't care that there's useless whitespace